### ISSUE TITLE
artsandculture.google.com 2.90 dosen't work

### ISSUE THREAD

**Issue by rokili** [lovasoa/dezoomify-rs#179]
https://artsandculture.google.com/asset/der-ungl%C3%A4ubige-thomas-michelangelo-merisi-named-caravaggio/OAEjjQkNdRL9sg

with wrong:
Unable to download tile 'https://lh3.googleusercontent.com/ci/ACqFvF1Ced6s2YtYB3_k70OIwZqQmGagJNPsEC34dK7GgIGym_JNVQCpwt1FOSLgvv_HD67hcqseoCA=x3-y0-z7-tkHpj2isisQbQ9jdh1TDckBsfGio'. Cause: network error: HTTP status client error (403 Forbidden) for url (https://lh3.googleusercontent.com/ci/ACqFvF1Ced6s2YtYB3_k70OIwZqQmGagJNPsEC34dK7GgIGym_JNVQCpwt1FOSLgvv_HD67hcqseoCA=x3-y0-z7-tkHpj2isisQbQ9jdh1TDckBsfGio

**Comment by jeffp1717** [comment]
Same issue.

[ETA:0s] ########################################   54/54   Unable to download tile 'https://lh3.googleusercontent.com/ci/ACqFvF2O7iWfG61bZU8XBGqiMPaYJKZ2MZWRfs4-xywNWUekJYpfLlY3wlV5LThPP2RgRGU5Yvl0Yw=x7-y5-z4-tM7UbZTL_BOFYbeorr8nV94HLXA0'. Cause: network error: HTTP status client error (403 Forbidden) for url (https://lh3.googleusercontent.com/ci/ACqFvF2O7iWfG61bZU8XBGqiMPaY

**Comment by londonuk371** [comment]
Yes get the same error with 2.9.0 and this error with 2.7.2 "ERROR Dezoomer error: Unable to create the dezoomer: Tried all of the dezoomers, none succeeded. They returned the following errors:"
I tried by changing my IP address but get the same error.

**Comment by lovasoa** [comment]
This should be fixed in v2.9.1 :)
